static/js/brand-store/SectionNav/SectionNav.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { Link, useParams } from "react-router-dom";
+import PropTypes from "prop-types";
+
+function SectionNav({ sectionName }) {
+  const { id } = useParams();
+
+  return (
+    <div className="p-tabs">
+      <ul className="p-tabs__list">
+        <li className="p-tabs__item">
+          <Link
+            to={`/admin/${id}/snaps`}
+            className="p-tabs__link"
+            aria-selected={sectionName === "snaps"}
```

Comment:
adding `role="tab"` to each link will make for a better experience for screen reader users (e.g. voiceover doesn't mention the status "selected" without it at all).